client/src/main.rs

```diff
@@ -623,6 +623,45 @@ fn main() {
                     Ok(())
                 }),
         )
+        .add_cmd(
+            Command::new("get-proof-of-attendances")
+                .description("creates proof of ProofOfAttendances for an <account> for the \
+                past <amount> ceremonies (which might be invalid, as reputation is not checked)")
+                .options(|app| {
+                    app.setting(AppSettings::ColoredHelp)
+                        .arg(
+                            Arg::with_name("accountid")
+                                .takes_value(true)
+                                .required(true)
+                                .value_name("SS58")
+                                .help("AccountId in ss58check format"),
+                        )
+                        .arg(
+                        Arg::with_name("amount")
```

Comment:
the naming "amount" seems odd to me.

People may not participate in subsequent ceremonies for the same cid. You can change location, you can skip ceremonies.

I would prefer to use the argument `ceremony-index` (`-i`) which can either be a positive absolute integer or a negative relative integer (-1 being the last ceremony) to precisely define a single ceremony index
Then you have to call this function for every proof, but that's nothing to worry about